modules/ossm-about-cluster-wide-migration.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: Concept
+[id="ossm-about-cluster-wide-migration_{context}"]
+= About cluster-wide migration
+
+During the migration process, two cluster-wide control planes run in the same cluster while the data plane namespaces are gradually migrated to the {SMProductName} 3.0 installation. One control plane is associated with the {SMProductName} {SMv2Version} installation and the other one is associated with the {SMProduct} 3.0 installation. You must carefully plan out the migration steps in order to avoid possible conflicts between the two control planes. The following topics describe the required conditions for successful migration.
+
+== Control planes and root certificate
+
+During the migration, the control plane associated with the 2.6 installation and the control plane associated with the 3.0 installation must share the same root certificate. This can be achieved by installing the 3.0 control plane to the same namespace as 2.6 control plane. The migration procedures show how to verify the root certificate is shared.
```

Comment:
```suggestion
During the migration, the control plane that is associated with the 2.6 installation and the control plane that is associated with the 3.0 installation must share a root certificate. This can be achieved by installing the 3.0 control plane into the same namespace as 2.6 control plane. The migration procedures show how to verify that the root certificate is shared.
```
or, as you use later:
```suggestion
During the migration, both control planes must share a root certificate. This can be achieved by installing the 3.0 control plane into the same namespace as 2.6 control plane. The migration procedures show how to verify that the root certificate is shared.
```

Reply:
done